packages/grafana-traces/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "author": "Grafana Labs",
+  "license": "Apache-2.0",
```

Comment:
Is this a license change if this is code from core? Would again probably opt to keep the same license as core for now.